### ISSUE TITLE
"Show next OTP" isn't working

### ISSUE THREAD

**Issue by sculpordwarfnix** [paolostivanin/OTPClient#234]
The new OTP Value shows pretty fast (like 1 second) and hides.

What do you think about:
-  the app automatically shows the next OTP value if the current one was displayed with a validity lower than 10 seconds?
- Keep always on the validity countdown?

**Comment by paolostivanin** [comment]
You're right, that shouldn't be happening. Every OTP should be shown for "validity" seconds.
